include/fty_common_db_library.h

```diff
@@ -79,6 +81,20 @@
 //  Opaque class structures to allow forward references
 //  These classes are stable or legacy and built in all releases
 //  Draft classes are by default not built in stable releases
+#ifdef FTY_COMMON_DB_BUILD_DRAFT_API
+typedef struct _fty_common_db_dbpath_t fty_common_db_dbpath_t;
```

Comment:
why do we have these defines? the strucs don't exist

Reply:
This is an autogenerated file.